hdate/__init__.py

```diff
@@ -491,11 +482,8 @@ def get_reading(self, diaspora):
                 else:
                     reading += 1
             if (JOIN_FLAGS[diaspora][self._h_year_type - 1][6] and
-                    (reading >= 51)):
```

Comment:
we don't need reading += 1 here?

Reply:
Same as above.

Impossible to reach this case.
Unittests cover all possible years. This case is never reached.